install.sh

```diff
@@ -33,6 +33,20 @@ chmod +x "${SHIM_PHPENV_RELOAD}"
 
 echo "%appwise ALL = (root) NOPASSWD: ${PHPENV_RELOAD}" > /etc/sudoers.d/phpenv-reload
 
+# set default PHP version
```

Comment:
I'd add a check (and message) here to see if there's already a file with a default version. Would be useful when we just want to install the latest version of these scripts, and not have to check what the previous default was.

Reply:
What should happen if this file already exists?
- show message + do not ask again for default version
- show message + ask again for default version
- ...